test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -969,18 +1016,15 @@ func executeHTTPRequest(method string, url string, body string) (string, error)
 	return string(respBody), nil
 }
 
-func enableAutoprovisioning() error {
-	enabled, err := isAutoprovisioningEnabled()
-	if err != nil {
-		glog.Errorf("Error: %s", err.Error())
-		return err
-	}
-	if enabled {
-		By("Autoprovisioning enabled.")
-		return nil
-	}
+func enableAutoprovisioning(resourceLimits string) error {
```

Comment:
Probably map for resources could work nicer, but no strong feelings.

Reply:
+1

Reply:
I don't think that it would be very helpful as we will not used this function in many places. Anyway there is {resource name, min , max} so I would have to create some structure containing min and max and then use it in a map. I think this is not worth.